prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -98,6 +98,7 @@ data class Hl7Configuration
     val valueSetOverrides: Map<String, ValueSet>? = emptyMap(),
     override val nameFormat: String = "standard",
     override val receivingOrganization: String?,
+    val convertPositiveDateTimeOffsetToNegative: Boolean? = false,
```

Comment:
Added a new feature flag to the HL7 translation configuration that controls the swapping of signs.